src/components/MyForums.js

```diff
@@ -151,40 +202,53 @@ const MyForums = () => {
     }*/
```

Comment:
remove the commented out code

Reply:
this commented out code was done by gabby, i think she wants to keep it there, not sure.

Reply:
@gabbytang do you need this code? should @c45choi remove it?

Reply:
I think this whole chunk of code is gone. Like I reworked the MyForums page from this point in my Edit Forums pr

Reply:
oh it is already gone? im rebasing so ill check rn